sns_ik_lib/test/sns_acc_ik_base_test.cpp

```diff
@@ -0,0 +1,151 @@
+/**  @file sns_acc_ik_base_test.cpp
+ *
+ *  @brief Unit Test: sns_acc_ik_base solver
+ *  @author Matthew Kelly
+ *
+ *    Copyright 2018 Rethink Robotics
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#include <Eigen/Dense>
+#include <ros/console.h>
+
+#include <sns_ik/sns_acc_ik_base.hpp>
+#include "rng_utilities.hpp"
+
+/*************************************************************************************************/
+
+void checkEqualVector(const Eigen::VectorXd& A, const Eigen::VectorXd& B, double tol)
```

Comment:
This function and the next one duplicate an existing utility function in the velocity test. These should both be moved to some test utilities header that can be accessed by any test. I suggest making a new file called "test_utilities" that lives next to "rng_utilities".